component-overview/examples/cards/GroupCard-custom.jsx

```diff
@@ -0,0 +1,11 @@
+import { GroupCard } from '@sb1/ffe-cards-react/src/GroupCard/GroupCard';
+
+() => {
+    return (
+        <GroupCard shadow={true}>
+            <div>Element nr 1 med egendefinert styling</div>
```

Comment:
Jag er lite osikker på om vi burde ha denne og op opmuntra til dette. Alltså selva exemplet. Att det går er vell en tin?

Reply:
Ja kanskje. Fra egen erfaring; hvis det ikke er et eksempel så visste jeg ikke at det gikk an.. Har for eksempel manuelt fjernet skyggen fra card, visste ikke at det fantes en shadow={false} før jeg kom til designsystemet! Det er jo lett å si at "man burde jo bare sjekke props", men når ikke jeg gjorde det så kan jeg tenke meg at minst en annen også ikke tenker på å gjøre det.. Prøver bare å tenke på de som nesten aldri rører designsystemet og er litt lengre fra det enn det vi er 😬 

MEN, jeg skulle si at flere av de eksemplene vi ser har noe på toppen som ser ut som må være custom ladg, men det kan faktisk passere som et kort det og. Skal prøve å legge til det som eksempel i stedet. Men fint å huske på til senere at om det ikke er et eksempel, så vet folk kanskje ikke at det går an..